frontend/src/metabase/components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -7,10 +7,19 @@ interface Props {
   className?: string;
   size?: number;
   borderWidth?: number;
+  "data-testid"?: string;
 }
 
-const LoadingSpinner = ({ className, size = 32, borderWidth = 4 }: Props) => (
-  <SpinnerRoot className={className} data-testid="loading-spinner">
+const LoadingSpinner = ({
+  className,
+  size = 32,
+  borderWidth = 4,
+  ...rest
```

Comment:
Why not `"data-testid": dataTestId`? this is used somewhere

Reply:
Yep I saw that, good call